Fix some PyTorch lints

r-barnes opened the pull request:
Summary: Fix some clang-tidy lints

Test Plan: Sandcastle

Differential Revision: D46152623

Skylion007 reviewed (approved):
Fix the linter error and looks good.

r-barnes committed:
Fix some PyTorch lints (#102413)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/102413

Fix some clang-tidy lints

Test Plan: Sandcastle

Reviewed By: jerryzh168

Differential Revision: D46152623

fbshipit-source-id: b66abc205265027e5abde8d1d1168b1ab2c3793f
